pkg/ui/workspaces/cluster-ui/package.json

```diff
@@ -91,41 +99,49 @@
     "babel-polyfill": "^6.26.0",
     "babel-preset-react-app": "^9.1.2",
     "chai": "^4.2.0",
-    "classnames": "^2.2.6",
+    "classnames": "2.3.2",
     "connected-react-router": "^6.8.0",
+    "copy-to-clipboard": "3.3.1",
+    "css-loader": "3.6.0",
     "d3-array": "^2.8.0",
     "d3-format": "^2.0.0",
     "d3-scale": "^3.2.3",
     "enzyme": "^3.11.0",
     "enzyme-adapter-react-16": "^1.15.2",
     "esbuild": "0.14.43",
     "esbuild-loader": "^2.19.0",
-    "eslint": "^6.8.0",
+    "eslint": "7.32.0",
     "eslint-config-prettier": "^6.10.0",
     "eslint-plugin-prettier": "^3.1.2",
     "eslint-plugin-react": "^7.18.3",
     "eslint-plugin-react-hooks": "^2.3.0",
     "fetch-mock": "^9.10.7",
     "file-loader": "^6.0.0",
     "highlight.js": "^10.2.0",
+    "history": "4.10.1",
     "http-proxy-middleware": "^1.0.5",
     "identity-obj-proxy": "^3.0.0",
+    "immer": "^9.0.3",
     "jest": "^27.5.1",
     "jest-canvas-mock": "^2.4.0",
     "jest-cli": "^27.5.1",
     "jest-environment-enzyme": "^7.1.2",
     "jest-enzyme": "^7.1.2",
     "jest-fetch-mock": "^3.0.3",
+    "jest-junit": "^16.0.0",
     "less": "^3.12.2",
-    "less-loader": "^6.2.0",
+    "less-loader": "5.0.0",
```

Comment:
We probably didn't mean to downgrade `less-loader`, right?

Reply:
The downgrade here solved the purpose of resolving the following inscrutable error. I couldn't tell you what it means, all I can tell you is that `less-loader@5.0.0` does not throw an error.

```
TypeError [ERR_INVALID_ARG_TYPE]: The "path" argument must be of type string. Received undefined
    at new NodeError (node:internal/errors:371:5)
    at validateString (node:internal/validators:120:11)
    at Object.normalize (node:path:1128:5)
    at new LessError (/private/var/tmp/_bazel_ricky/be70b24e7357091e16c49d70921b7985/sandbox/darwin-sandbox/863/execroot/com_github_cockroachdb_cockroach/bazel-out/darwin_arm64-fastbuild/bin/pkg/ui/workspaces/cluster-ui/node_modules/.aspect_rules_js/less-loader@6.2.0_webpack_4.46.0/node_modules/less-loader/dist/LessError.js:15:169)
    at /private/var/tmp/_bazel_ricky/be70b24e7357091e16c49d70921b7985/sandbox/darwin-sandbox/863/execroot/com_github_cockroachdb_cockroach/bazel-out/darwin_arm64-fastbuild/bin/pkg/ui/workspaces/cluster-ui/node_modules/.aspect_rules_js/less-loader@6.2.0_webpack_4.46.0/node_modules/less-loader/dist/index.js:62:14 {
  code: 'ERR_INVALID_ARG_TYPE'
}

```